### DISCUSSION TITLE
MMM01

### DISCUSSION THREAD

**Discussion by gatarib** [sanni/cartreader#372] [General Chat]
I tried dumping Momotaro Collection 2, which is MMM01 for Game Boy, but originally the selection screen for momotaro dengeki2 and momotaro densetsu gaiden appears (image), but I cannot make the selection and only momotaro dengeki2 starts up
![無題](https://user-images.githubusercontent.com/77560501/170815583-cf09d73e-d353-4d29-83c8-e636f01a14a7.jpg)
.

**Comment by mariomadproductions** [comment]
What size in bytes and sha256 do you get for your ROM?
